README.md

```diff
@@ -88,21 +88,31 @@ HTTP(S) proxies can be configured through any of the following methods, in prece
 * The `HTTPS_PROXY` and/or `HTTP_PROXY` environment variables, in the same URL format as
   `BALENARC_PROXY`.
 
-> Note: The `balena ssh` command has additional setup requirements to work behind a proxy.  
-> Check the [installation instructions](https://github.com/balena-io/balena-cli/blob/master/INSTALL.md).
-
-Some installations of the balena CLI also include support for the `BALENARC_NO_PROXY` environment
-variable, which allows proxy exclusion patterns to be defined. The current support status is listed
-below. Eventually, all installation types will have support for it.
-
-OS | Installation type | BALENARC_NO_PROXY environment variable support
--- | ----------------- | ----------------------------------------------
-Windows | standalone zip | Supported with CLI v11.24.0 and later
-Windows | native/GUI     | Not supported
-macOS   | standalone zip | Not supported
-macOS   | native/GUI     | Supported with CLI v11.24.0 and later
-Linux   | standalone zip | Not supported
-Any     | npm            | Supported with Node.js >= v10.16.0 and CLI >= v11.24.0
+> Note: The `balena ssh` command has additional setup requirements to work behind a proxy.
+> Check the [installation instructions](https://github.com/balena-io/balena-cli/blob/master/INSTALL.md),
+> and ensure that the proxy server is configured to allow proxy requests to ssh port 22, using
+> SSL encryption. For example, in the case of the [Squid](http://www.squid-cache.org/) proxy
+> server, it should be configured with the following rules in the `squid.conf` file:  
+> `acl SSL_ports port 22`  
+> `acl Safe_ports port 22`  
+
+#### Proxy exclusion
+
+The `BALENARC_NO_PROXY` variable may be used to exclude specified destinations from proxying.
+
+> * This feature requires balena CLI version 11.30.7 or later. In the case of the npm [installation
+>   option](https://github.com/balena-io/balena-cli/blob/master/INSTALL.md), it also requires
```

Comment:
This is a correct but simplified statement. Earlier CLI versions between 11.24.0 and 11.30.7 added support for some combinations of installation type and platform (standalone zip / GUI installer / npm, **x** Windows / macOS / Linux). It all depends on the version of Node.js that gets bundled with the installer (BALENARC_NO_PROXY requires Node.js v10.16.0 or later). Two significant events took place since I had last updated these proxy instructions:
* [pkg](https://www.npmjs.com/package/pkg) updated "the Node 10 version" that they patch and bundle in the single-file binary from 10.15.3 to 10.17.0. I think this took place around early March, and I had not even realised.
* Last week, Page updated the host OS Node.js in the balena CI Windows worker from 8.15.1 to 10.19.0.